code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -98,14 +98,48 @@
 	max_storage_space = ITEMSIZE_COST_NORMAL * 25
 	max_w_class = ITEMSIZE_NORMAL
 	can_hold = list(/obj/item/weapon/ore)
-	var/stored_ore = list()
+	var/current_capacity = 0
+	var/max_pickup = 100 //How much ore can be picked up in one go. There to prevent someone from walking on a turf with 10000 ore and making the server cry.
+	var/list/stored_ore = list(
+		"sand" = 0,
+		"hematite" = 0,
+		"carbon" = 0,
+		"raw copper" = 0,
+		"raw tin" = 0,
+		"void opal" = 0,
+		"painite" = 0,
+		"quartz" = 0,
+		"raw bauxite" = 0,
+		"phoron" = 0,
+		"silver" = 0,
+		"gold" = 0,
+		"marble" = 0,
+		"uranium" = 0,
+		"diamond" = 0,
+		"platinum" = 0,
+		"lead" = 0,
+		"mhydrogen" = 0,
+		"verdantium" = 0,
+		"rutile" = 0)
 	var/last_update = 0
 
 /obj/item/weapon/storage/bag/ore/holding
 	name = "mining satchel of holding"
 	desc = "Like a mining satchel, but when you put your hand in, you're pretty sure you can feel time itself."
 	icon_state = "satchel_bspace"
-	max_storage_space = ITEMSIZE_COST_NORMAL * 75 // 3x
+	max_storage_space = ITEMSIZE_COST_NORMAL * 15000 // This should never, ever, ever be reached.

```

Comment:
Is there a reason to not use the `INFINITY` define here?

Reply:
I could use INFINITY but I'm weary of making something in the game _actually_ infinite as it could interact poorly with something in the future.